pkg/ipamd/ipamd.go

```diff
@@ -134,6 +134,10 @@ const (
 	// disableLeakedENICleanup is used to specify that the task checking and cleaning up leaked ENIs should not be run.
 	envDisableLeakedENICleanup = "DISABLE_LEAKED_ENI_CLEANUP"
 
+	// disableENIMetadataReconcile is used to specify that EC2 API calls should be skipped and only IMDS metadata should be
+	// used to get ENI information.
+	envDisableENIMetadataReconcile = "DISABLE_ENI_METADATA_RECONCILE"
```

Comment:
Lets name it as `DISABLE_EC2_API_CALLS` instead..

Reply:
Renamed to ```ENABLE_IMDS_ONLY_MODE``` but let me know which name you feel better